src/org/labkey/test/components/ui/files/AttachmentCard.java

```diff
@@ -144,15 +145,29 @@ public static class FileAttachmentCardFinder extends WebDriverComponentFinder<At
     {
         private final Locator.XPathLocator _baseLocator = Locator.tagWithClass("div", "attachment-card");
         private String _fileName = null;
+        private String _title = null;
 
         public FileAttachmentCardFinder(WebDriver driver)
         {
             super(driver);
         }
 
-        public FileAttachmentCardFinder withFile(String fileName)
```

Comment:
Initially, the `withFile `method caused finder to find the `AttachmentCard `by its title attribute (based on the assumption that it would always be the file).  I discovered during this story that the title could include more info, such as the hierarchy of location in the dav folders, so I changed the existing function (and refactored its usages) to be `withTitle`.

The reason for this is I needed to find `AttachmentCards `by just their file name, and it's right there in the attachment-card__name element, so I implemented another method to do that and named it `withFileName`